examples/pooling_example.py

```
@@ -0,0 +1,293 @@
+    BatchGlobalPooling,
+    GlobalPooling,
+    Set2Set,
+)
+
+
+def create_sample_graph(
+    num_nodes=50, num_features=16, num_edges=100
+) -> tuple[np.ndarray, np.ndarray]:
+    """
+    Generates a sample graph with random node features and edge indices.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)